roles/kubernetes/master/handlers/main.yml

```diff
@@ -93,15 +93,17 @@
 
 - name: Master | wait for kube-scheduler
   uri:
-    url: http://localhost:10251/healthz
+    url: https://localhost:10259/healthz
+    validate_certs: no
   register: scheduler_result
```

Comment:
I assume we can use kube_apiserver_client_cert/kube_apiserver_client_key to validate both scheduler and controller manager endpoints, because they point to `{{ kube_cert_dir }}/ca.crt` by default but I'm wondering if that's a good solution to rely on a default that people can change.

Reply:
I tried to validate the url using certs but seems maybe the ca is needed or another issue as I wasn't able to query using the correct certs (even the API Server check doesn't work without the 'validate certs' set to no

Regarding the default you mean the ports? If so indeed but as we don't have a kubespray variable for that we don't have to worry I'd say.. Maybe we can add a variable and use it here with 10259 as the default